_schemaV2.graphql

```diff
@@ -1279,8 +1279,10 @@ type Artwork implements Node & Searchable & Sellable {
   description(format: Format): String
   dimensions: dimensions
   displayLabel: String
+  displayPriceRange: Boolean
```

Comment:
We need this as well so we can adjust the form if you're making an offer on a work with a price range.